docs/apps/rapids.md

```diff
@@ -81,14 +81,14 @@ list-packages
 
     Note that login nodes are not intended for heavy computing, please use
     slurm batch jobs instead. See our [instructions on how to use the batch job
-    system](../computing/running/getting-started.md).
+    system](../computing/running/index.md).
 
 ### Local storage
 
 The GPU nodes have fast local storage which is useful for IO-intensive
 applications. See our [general instructions on how to take the fast local
 storage into
-use](../computing/running/creating-job-scripts-puhti.md#local-storage).
+use](../computing/running/creating-job-scripts-puhti/#local-storage).
```

Comment:
link broken

Reply:
Another case of a link split on two lines. I'll try to come up with a way to find all of these.

Reply:
Thanks! I think this would be valuable since I fear it is quite common practice to split links.. I mean why not, it is valid markdown syntax and helps to keep the content readable in a text editor.

Reply:
I think there are around 200 of them.

Reply:
Looks like the "checkbuild" test catches two-line links if they are to a `.md` file that doesn't exist. This is because MkDocs throws a warning and "checkbuild" doesn't allow that.

Reply:
148cac0 fixes the link in question.